homework/Stupns/homework6.py

```diff
@@ -0,0 +1,28 @@
+import requests
+import os.path
+
+
+url = 'https://raw.githubusercontent.com/aodarc/LITS009/master/lections/4.recurtion_example.py'
+
+
+def download(url):
+    new = requests.get(url, allow_redirects=True)
+    if new.status_code == 404:
+        print('No such file found at %s' % url)
+        return
+    if not url.rfind('.py'):
+        a = print('File extension with .py not found')
```

Comment:
in this case if you want return None, you should skip `a` variable and use just `return`